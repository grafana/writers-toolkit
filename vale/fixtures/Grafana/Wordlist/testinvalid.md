# Promtail

Don't use:

- promtail

# Data Firehose

Don't use:

- Kinesis Data Firehose
- Firehose
- Kinesis Firehose
