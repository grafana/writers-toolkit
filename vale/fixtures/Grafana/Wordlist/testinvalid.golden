testinvalid.md:5:3:Grafana.Wordlist:Use 'Promtail' instead of 'promtail'.
testinvalid.md:11:3:Grafana.Wordlist:Use 'Data Firehose' instead of 'Kinesis Data Firehose'.
testinvalid.md:12:3:Grafana.Wordlist:Use 'Data Firehose' instead of 'Firehose'.
testinvalid.md:13:3:Grafana.Wordlist:Use 'Data Firehose' instead of 'Kinesis Firehose'.
